src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx

```diff
@@ -142,7 +143,7 @@ const AuthenticatorButton: UISchemaElementComponent<{
           id={`${iconName}-label`}
           sx={{ fontSize: '1rem', margin: 0, textAlign: 'start' }}
           data-se="authenticator-button-label"
-          className="authenticator-label"
+          className="authenticator-label no-translate"
```

Comment:
I think this is the root cause of each authenticator button label having the `no-translate` class. I think this should be conditional and only apply to things that cannot be translated like Yubikey / Okta verify / Symantec VIP / etc